core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -52,6 +73,14 @@ public MissingQueryBuilder nullValue(boolean nullValue) {
     }
 
     /**
+     * Should the missing filter automatically include fields with null value configured in the
+     * mappings.
+     */
```

Comment:
Maybe something along the lines of "Returns true if ..." would make the wording a bit clearer?